README.md

```diff
@@ -31,7 +31,7 @@ There are two ways to get a source tree:
   Note: cloning GHC from Github requires a special setup. See [Getting a GHC
   repository from Github] [7].
 
-  **DO NOT submit pull request directly to the github repo.**
+  We will accept easily-reviewed documentation changes as GitHub Pull Requests.
```

Comment:
Oh, right, I had overlooked that bit.

Well, on the proposal discussion, quchen mentions ["housekeeping"](https://github.com/ghc-proposals/ghc-proposals/pull/11#issuecomment-269816034) changes that he would like to submit on Github, and which I think should be encouraged too. And if a PR can't easily be handled on GitHub, there's always the option of asking the submitter to move his patch to phabricator.

But let's see what @bgamari or possibly other maintainers are thinking about this.